client/src/components/GameInvitation.js

```diff
@@ -0,0 +1,64 @@
+/*
+Component for inviting friends to a new game.
+*/
+
+import React, { useState, useEffect } from "react";
+import { Grid, Button } from '@material-ui/core';
+import LinkIcon from '@material-ui/icons/Link';
+
+function GameInvitation() {
+
+    // States for user input and list of emails invited.
+    const [emailInput, setEmailInput] = useState("");
+    const [emailList, setEmailList] = useState([]);
+
+    // Updates the state when the user input changes.
+    const onInputChange = (e) => {
+        setEmailInput(e.target.value);
+    }
+
+    // Add the input to the list of emails then clear the input.
+    // Need to implement actual sending of the email.
+    const sendInvitation = () => {
+
+        // Only if the user input is not empty.
+        if (emailInput.length > 0) {
+            setEmailList([...emailList, emailInput]);
+            setEmailInput("");
+
+            // Implement sending of the actual invitation here.
+        }
+    }
+
+    return (
+        <div>
+            <Grid container alignItems="center">
+                <Grid item xs={9}>
+                    <div className="item item-left">
+                        Invite friends via email:
+                        <form>
```

Comment:
If you're using a form wrapper for this, you should include an onSubmit handler. Probably you can just change the name of your handleClick function and then change your button to type="submit". Be sure to prevent the default behavior for the submit event though

Reply:
Made these changes